Makefile

```diff
@@ -69,28 +70,32 @@ code/fix:
 
 .PHONY: image/build
 image/build: code/compile
-	@operator-sdk build $(REG)/$(ORG)/$(PROJECT):v$(TAG)
+	@operator-sdk build $(INTEGREATLY_OPERATOR_IMAGE)
 
 .PHONY: image/push
 image/push:
-	docker push $(REG)/$(ORG)/$(PROJECT):v$(TAG)
+	docker push $(INTEGREATLY_OPERATOR_IMAGE)
 
 .PHONY: image/build/push
 image/build/push: image/build image/push
 
 .PHONY: image/build/test
 image/build/test:
-	operator-sdk build --enable-tests $(REG)/$(ORG)/$(PROJECT):v$(TAG)
+	operator-sdk build --enable-tests $(INTEGREATLY_OPERATOR_IMAGE)
 
 .PHONY: test/unit
 test/unit:
 	TEMPLATE_PATH=$(TEMPLATE_PATH) ./scripts/ci/unit_test.sh
 
+.PHONY: prow/test/e2e
```

Comment:
I think we need to _either_ get prow to use this target, _or_ remove this and set the `INTEGREATLY_OPERATOR_IMAGE` env var in the prow config.

I'm not sure which is better, or if there are better alternatives. @philbrookes @mikenairn thoughts?

Reply:
We can change the prow config to use this target instead https://github.com/openshift/release/blob/master/ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml#L57. As per the jira i linked, we planned to do that anyway https://issues.redhat.com/browse/INTLY-4210

Main issue i see with this that we will not get any logs on failure so figuring out why something has failed will become more difficult unless we can gather operator logs and store them in the build artifacts.